torch/_inductor/graph.py

```diff
@@ -356,6 +356,11 @@ def call_function(self, target, args, kwargs):
         if target is operator.getitem and isinstance(args[0], (list, tuple)):
             return super().call_function(target, args, kwargs)
 
+        # TODO (tmanlaibaatar) Skip now because we don't have
+        # inductor lowering for this op.
+        if target == torch.ops.aten._assert_async.msg:
```

Comment:
It would be cleaner to do this in `decomposition.py`

Reply:
yep! It just decomposes to nothing now